src/back_end/services/StatisticService.js

```diff
@@ -114,18 +117,132 @@ module.exports = (function () {
                         }
                     }]
                 };
-                dataBaseService.setDataToDB(urlStatisticsDataDB, 'Day_Statistics', result);
+                dataBaseService.setDataToDB(urlStatisticsDataDB, 'City_Day_Statistics', result);
             }, function (err) {
                 console.error('Data is not collected!\n', err, err.stack);
                 logger.logError(err);
             });
         });
     },
 
+        dayStatistics = function (serchTime) {
+            var start = new Date(serchTime.getTime()),
+                end = new Date(serchTime.getTime());
+            start.setHours(0, 0, 0, 0);
+            end.setHours(23, 59, 59, 999);
+            _.each(config.getServicesNames(), function (service) {
+                dataBaseService.getStatisticsOnServices(urlWeatherDataDB, 'unifiedWeather',
+                    parseInt(start.getTime() / 1000, 10), parseInt(end.getTime() / 1000, 10), service).then(function (dataArr) {
+                    console.info('Data services successfully collected!');
+
+                    var lowestTempObj = {},
+                        highesTempObj = {},
+                        lowestHumObj = {},
+                        highesHumObj = {},
+                        lowestWindSpeedObj = {},
+                        highesWindSpeedObj = {},
+                        result = {},
+                        avgTempVal = 0,
+                        avgHumVal = 0,
+                        avgWindSpeedVal = 0,
+                        lowestTemp = Number.POSITIVE_INFINITY,
+                        lowestHum = Number.POSITIVE_INFINITY,
+                        lowestWindSpeed = Number.POSITIVE_INFINITY,
+                        highestTemp = Number.NEGATIVE_INFINITY,
+                        highestHum = Number.NEGATIVE_INFINITY,
+                        highestWindSpeed = Number.NEGATIVE_INFINITY;
+
+                    _.each(dataArr, function (data) {
+                        avgTempVal += data.temp;
+                        avgHumVal += data.humidity;
+                        avgWindSpeedVal += data.temp;
+                        if (data.temp < lowestTemp) {
+                            lowestTemp = data.temp;
+                            lowestTempObj = {
+                                'temp': data.temp,
+                                'cords': data.coords
+                            };
+                        }
+                        if (data.temp > highestTemp) {
+                            highestTemp = data.temp;
+                            highesTempObj = {
+                                'temp': data.temp,
+                                'cords': data.coords
+                            };
+                        }
+                        if (data.humidity < lowestHum) {
+                            lowestHum = data.humidity;
+                            lowestHumObj = {
+                                'temp': data.humidity,
```

Comment:
Fix this please - name of object field must be the saim as parameter. For example there must be 'hum' : data.humidity.